src/main/java/ui/components/issuepicker/IssueCard.java

```diff
@@ -0,0 +1,176 @@
+package ui.components.issuepicker;
+
+import java.util.Optional;
+
+import backend.resource.TurboIssue;
+import backend.resource.TurboLabel;
+import backend.resource.TurboMilestone;
+import backend.resource.TurboUser;
+import javafx.beans.binding.Bindings;
+import javafx.geometry.Insets;
+import javafx.geometry.Pos;
+import javafx.scene.control.Label;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.Background;
+import javafx.scene.layout.BackgroundFill;
+import javafx.scene.layout.CornerRadii;
+import javafx.scene.layout.FlowPane;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.VBox;
+import javafx.scene.paint.Color;
+import javafx.scene.text.Text;
+import ui.GuiElement;
+
+/**
+ * Represents an individual issue card not linked to any panel
+ */
+public class IssueCard extends VBox {
```

Comment:
How much of this is shared with ListPanelCard? Can we factor out the common parts and share as much as possible? So it doesn't get out of sync.